bin/public/installDocker.py

```diff
@@ -38,6 +39,17 @@ def install_docker(args):
     version_obj = version.Version("Installdocker", SCRIPT_VERSION)
     version_obj.check_executed()
 
+    proxy_host = config.general.get_proxy_host()
+    proxy_port = config.general.get_proxy_port()
+
+    #Prepare proxy access for YUM
+    proxy_user=raw_input('Enter SQUID username: ')
```

Comment:
Hmm, I don't think stopping a syco installation to request a username/password from stdin is a good idea. It breaks the automation and will not work with remote-install etc. Also, not all environments have a proxy in place so any proxy-specific stuff should be inside of an if-block that's only executed if the proxy-host and proxy-port are set (this if block is already on line 60.